src/main/java/org/apache/gossip/crdt/CrdtModule.java

```diff
@@ -42,6 +42,13 @@
   @JsonIgnore abstract boolean isEmpty();
 }
 
+abstract class GrowOnlyCounterMixin{
+  @JsonCreator
+  GrowOnlyCounterMixin(@JsonProperty("myID") String myID, @JsonProperty("counters") Map<String , Long> count){ }
```

Comment:
I do not think we should serialize myId. The reason: if one node sets myID and sends that data to another system myId is now incorrect. We need some way to properly hydrate this.